test/evaporate.spec.js

```diff
@@ -3,7 +3,7 @@ import chaiSinon from 'sinon-chai'
 import sinon from 'sinon'
 import test from 'ava'
 
-import Evaporate from '../codemod';
+import Evaporate from '../dist/bundle'
```

Comment:
Are we sure running tests against built dist bundle is a good idea? Or will we improve this later?

Reply:
I've improved this by aliasing the bundle using module-alias. Unfortunately, we need to do this in this way as ava can't handle TypeScript imports on JS files, unless I put the ts-node register on it, which makes the tests to run much slower and it breaks sinon, chai, etc...

So I've opted to go through the route of precompiling the code, as you can see in the improved version.

Reply:
Yeah, I can see, but I don't think this is the right way to go. It's also pretty slow, compiling stuff "explicitly" before running tests. Maybe `ava` doesn't have such options, but `jest` for sure can compile the ts files itself without extra webpack config and without running agains dist. The extra dependency could be spared.

Github Actions are great, let's go with that instead of Travis. But it should pass.

Reply:
Yes, I totally agree with the jest thing. Apparently there's a cool codemod that automagically changes the tests to jest:
https://jestjs.io/docs/en/migration-guide

I'll try that

Reply:
Unfortunately, it won't be possible to convert those tests to Jest. They rely a lot on passing data over the context of the tests, and because of that, the conversion is next to impossible given the number of tests.

I'd really like to use Jest on this project, as it's my favorite test runner/framework, but it won't be possible at least for now.